x-pack/legacy/plugins/siem/public/store/network/reducer.ts

```diff
@@ -71,6 +71,14 @@ export const initialNetworkState: NetworkState = {
         },
         isPtrIncluded: false,
       },
+      [NetworkTableType.tls]: {
+        activePage: DEFAULT_TABLE_ACTIVE_PAGE,
+        limit: DEFAULT_TABLE_LIMIT,
+        tlsSortField: {
+          field: TlsFields._id,
+          direction: Direction.desc,
+        },
+      },
```

Comment:
In this reducer, the actions for `updateTlsSort` and `updateTlsLimit` need to check which `networkType` to update, much like it is done in `updateTopNFlowSort`